gerador-boletos/src/test/java/com/tdd/BoletoTests.java

```diff
@@ -0,0 +1,22 @@
+package com.tdd;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Date;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class BoletoTests {
+    @Test
+    @DisplayName("Testa se o pagamento esta sendo criado com sucesso")
```

Comment:
Nome da entidade errado
```suggestion
    @DisplayName("Testa se o boleto esta sendo criado com sucesso")
```